apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -1293,7 +1277,9 @@ const sendMessageEpic: AppEpic = (action$, state$) =>
                 getNewConversationName(payload.conversation, payload.message),
                 conversations.filter(
                   (conv) =>
-                    conv.folderId === payload.conversation.folderId &&
+                    (conv.folderId === payload.conversation.folderId ||
+                      (isEntityIdLocal(payload.conversation) &&
+                        conv.folderId === conversationRootFolderId)) &&
```

Comment:
```suggestion
                    (conv.folderId === payload.conversation.folderId &&
                      !isEntityIdLocal(payload.conversation)) &&
```

I suppose we don't need to check local conversation names since them not exist?